client/src/components/App.jsx

```diff
@@ -1,20 +1,112 @@
 import React from 'react';
+import axios from 'axios';
+import TabNav from './TabNav';
+import TopNav from './TopNav';
+import Cohort from './Cohort';
+import TeamList from './TeamList';
 
 export default class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      initial: [],
+      sprintCohorts: [
```

Comment:
Let's keep this an MVP approach right now but consider maintenance of these sprint/team/etcCohorts at a good reflection point.  I think it reasonably incurs some technical debt.  

The question:  Is there a way to simplify management of the awareness of the system what "phase" of the program cohorts are in.